presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestTablePartitioningInsertInto.java

```diff
@@ -0,0 +1,121 @@
+import com.teradata.tempto.query.QueryResult;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.tests.TestGroups.HIVE_CONNECTOR;
+import static com.facebook.presto.tests.TestGroups.SMOKE;
+import static com.teradata.tempto.Requirements.compose;
+import static com.teradata.tempto.fulfillment.table.hive.InlineDataSource.createResourceDataSource;
+import static com.teradata.tempto.query.QueryExecutor.query;
+import static com.teradata.tempto.query.QueryType.UPDATE;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestTablePartitioningInsertInto
+        extends HivePartitioningTest
+        implements RequirementsProvider
+{
+    @Inject
+    private MutableTablesState mutableTablesState;
+
+    private static final String PARTITIONED_NATION_NAME = "partitioned_nation_read_test";
+    private static final String TARGET_NATION_NAME = "target_nation_test";
+
+    private static final int NUMBER_OF_LINES_PER_SPLIT = 5;
+    private static final String DATA_REVISION = "1";
+    private static final HiveTableDefinition PARTITIONED_NATION =
+            HiveTableDefinition.builder()
+                    .setName(PARTITIONED_NATION_NAME)
+                    .setCreateTableDDLTemplate("" +
+                            "CREATE EXTERNAL TABLE %NAME%(" +
+                            "   p_nationkey     INT," +
+                            "   p_name          STRING," +
+                            "   p_comment       STRING) " +
+                            "PARTITIONED BY (p_regionkey INT)" +
+                            "ROW FORMAT DELIMITED FIELDS TERMINATED BY '|' ")
+                    .addPartition("p_regionkey=1", createResourceDataSource(PARTITIONED_NATION_NAME, DATA_REVISION, partitionDataFile(1)))
+                    .addPartition("p_regionkey=2", createResourceDataSource(PARTITIONED_NATION_NAME, DATA_REVISION, partitionDataFile(2)))
+                    .addPartition("p_regionkey=3", createResourceDataSource(PARTITIONED_NATION_NAME, DATA_REVISION, partitionDataFile(3)))
+                    .build();
+
+    private static final HiveTableDefinition TARGET_NATION =
+            HiveTableDefinition.builder()
+                    .setName(TARGET_NATION_NAME)
+                    .setCreateTableDDLTemplate("" +
+                            "CREATE EXTERNAL TABLE %NAME%(" +
+                            "   p_nationkey     INT," +
+                            "   p_name          STRING," +
+                            "   p_comment       STRING," +
+                            "   p_regionkey     INT) " +
+                            "ROW FORMAT DELIMITED FIELDS TERMINATED BY '|' " +
+                            "LOCATION '%LOCATION%'")
+                    .setNoData()
+                    .build();
+
+    private static String partitionDataFile(int region)
+    {
+        return "com/facebook/presto/tests/hive/data/partitioned_nation/nation_region_" + region + ".textfile";
```

Comment:
Where is this file read from? Is this relative to the classpath, to a specific directory, etc?